build/synapse/Dockerfile

```diff
@@ -3,14 +3,17 @@ LABEL maintainer="Raiden Network Team <contact@raiden.network>"
 
 ARG SYNAPSE_VERSION
 
```

Comment:
DO we still need this if we build from latest master?

Reply:
My change is temporary so i didn't cleanup what was there before. i expect my changes to be reverted once we have a synapse release which includes the fix we're trying to test.